h1-lwt.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "h1-lwt",
+  "version": "1.0.0",
+  "license": "BSD-3-clause",
+  "homepage": "https://github.com/reason-native-web/h1",
+  "bugs": { "url": "https://github.com/reason-native-web/h1/issues" },
+  "esy": {
+    "build": "dune build --only-packages=httpaf-lwt --root=./httpaf --profile=release -j 4",
+    "install": "esy-installer #{self.target_dir / 'default' / 'httpaf-lwt.install'}"
+  },
+  "dependencies": {
+    "ocaml": ">=4.3.0",
+    "@opam/dune": ">=1.5.0",
+    "h1": ">=1.0.0",
```

Comment:
I think I'll edit the constraints to `1.0.x`? What do you think? I don't want weird build errors in the future :D.